pkg/deploy/deploy.go

```diff
@@ -240,6 +249,27 @@ func (deploy *Deployer) UninstallOLM() error {
 			return fmt.Errorf("failed to uninstall the %s metering namespace: %v", deploy.config.Namespace, err)
 		}
 	}
+	if deploy.config.DeleteCRB {
+		err = deploy.uninstallMeteringRole()
+		if err != nil {
+			return fmt.Errorf("failed to delete the metering role: %v", err)
+		}
+
+		err = deploy.uninstallMeteringRoleBinding()
+		if err != nil {
+			return fmt.Errorf("failed to delete the metering role binding: %v", err)
+		}
```

Comment:
I realize this is probably not needed as OLM will create these resources for us, and mark the either the Subscription or CSV resources as the owner reference, so deleting those resources will implicitly delete these roles/rolebindings.